### ISSUE TITLE
[enhancement] --only-cities : just give cities name with no additional chars

### ISSUE THREAD

**Issue by Euzebius** [p0dalirius/GeoWordlists#9]
Hy P0da,

Great tool as always!
I was thinking that maybe a switch like --only-cites would make sense for people that would use this dictionary to crack password with rules. This would prevent some duplicates and free some GPU time.

Like for instance you only get `Rouen` instead of: 
```
Rouen76
Rouen76!
Rouen76000
Rouen76000!
Rouen76100
Rouen76100!
```

Another option would be to like offer a companion script to only grab the alpha chars (I can provide this as a PR if it's easier for the integration maybe)

**Comment by p0dalirius** [comment]
Hey!

Thank you! I like the idea!

I am working on a set of yml files to define the derivation rules for the v2.0 release. 

I like the new option you proposed although I would prefer to name it `--no-derivation` or `--no-variant` in light of the future v2.0 release

**Comment by Euzebius** [comment]
No problem, totally makes sense.
I will modify my pull request accordingly to help for this.

**Comment by p0dalirius** [comment]
Fixed in #11
